pkg/importer/dataStream.go

```diff
@@ -897,14 +904,23 @@ func (d *DataStream) convertQcow2ToRaw(src, dest string) error {
 
 // Copy endpoint to dest based on passed-in reader.
 func (d *DataStream) copy(dest string) error {
-	if util.GetAvailableSpace(d.ScratchDataDir) > int64(0) {
+	if util.GetAvailableSpace(d.ScratchDataDir) > int64(0) && d.VolumeMode != v1.PersistentVolumeBlock {
```

Comment:
Again, having to clean up the scratch space is not affected by the target being block or not, we have to clean it up regardless.